sway/tree/root.c

```diff
@@ -62,8 +62,13 @@ void root_scratchpad_add_container(struct sway_container *con) {
 	struct sway_container *parent = con->parent;
 	struct sway_workspace *workspace = con->workspace;
 	container_set_floating(con, true);
-	container_floating_set_default_size(con);
-	container_floating_move_to_center(con);
+
+	// When a tiled window is sent to scratchpad, center and resize it.
+	if (!container_is_floating(con)) {
```

Comment:
This will always be true since it is being set to floating right before the check. The `container_set_floating` call needs to be moved inside.

Reply:
oops, I guess that's what I get for being tired and coding.

Reply:
LGTM as soon as this change is pushed